pkgs/tools/system/runitor/default.nix

```diff
@@ -1,30 +1,21 @@
-{ lib, buildGoModule, fetchFromGitHub, testers, runitor }:
+{ lib, buildGoModule, fetchFromGitHub, fetchpatch, testers, runitor }:
```

Comment:
I already fixed this, see commit https://github.com/NixOS/nixpkgs/pull/178897/commits/9f832703fca7f91fd810a1e9269e1cdb1833e4d6

Reply:
I made this suggestion before yours. I'm not suggesting any change here. When I approved PR, it showed it this way. Some GitHub glitch?

Reply:
> And now, I don't know know why this suggestion has been 'updated' to current date. Because this is an old suggestion.

Something that GitHub does is you can do a comment and not publish if you don't review it. And them the comment only appears after you review.

I think this is what happened here, your comment was only shown after you approved this PR.

Reply:
Seems so.